diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 631c30c8f153e72cd92de457d8ba6f44574edbe6..9b6acafa5cc1c133ae3c4266147921c0251df0ae 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -738,6 +738,7 @@ class Browser : public TabStripModelObserver,
   void RunScreenAIAnnotator();
 #endif
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
