diff --git a/chrome/browser/net/stub_resolver_config_reader.cc b/chrome/browser/net/stub_resolver_config_reader.cc
index 1837acea1ef050f0d8245bd7a968a57315685522..4b112c7da0d5148bd59f0a2d4c01fb41cb17e3bd 100644
--- a/chrome/browser/net/stub_resolver_config_reader.cc
+++ b/chrome/browser/net/stub_resolver_config_reader.cc
@@ -372,6 +372,7 @@ SecureDnsConfig StubResolverConfigReader::GetAndUpdateConfiguration(
   if (secure_dns_mode != net::SecureDnsMode::kOff) {
     doh_config = net::DnsOverHttpsConfig::FromStringLax(
         local_state_->GetString(prefs::kDnsOverHttpsTemplates));
+    BRAVE_GET_AND_UPDATE_CONFIGURATION
   }
   if (update_network_service) {
     content::GetNetworkService()->ConfigureStubHostResolver(
