diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index 60795f5e596c96330201c97dc1eef9da200ac099..cd96f8c3b2aea92f308be07ec3236e82921ab9ea 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -730,6 +730,7 @@ class Browser : public TabStripModelObserver,
   void RunScreenAIAnnotator();
 #endif
 
+  BRAVE_BROWSER_H
  private:
   friend class BrowserTest;
   friend class ExclusiveAccessTest;
