diff --git a/net/dns/resolve_context.h b/net/dns/resolve_context.h
index d9c0e756e9d9c6a5d628c149a6218993cb528981..df23fa3f53c2a6f3375be2341e4271a41fb633f6 100644
--- a/net/dns/resolve_context.h
+++ b/net/dns/resolve_context.h
@@ -194,6 +194,7 @@ class NET_EXPORT_PRIVATE ResolveContext : public base::CheckedObserver {
     return weak_ptr_factory_.GetWeakPtr();
   }
 
+  BRAVE_RESOLVE_CONTEXT_H
  private:
   friend DohDnsServerIterator;
   friend ClassicDnsServerIterator;
