diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
index 77a5c44a20ae776487d3b8bb0ddbc4c6fb54287f..5456848d0d55b208c327237886e0178b95f51760 100644
--- a/net/dns/dns_transaction.cc
+++ b/net/dns/dns_transaction.cc
@@ -1364,6 +1364,7 @@ class DnsTransactionImpl : public DnsTransaction,
     DCHECK(secure_);
 
     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
+    BRAVE_MAKE_HTTP_ATTEMPT
 
     unsigned attempt_number = attempts_.size();
     ConstructDnsHTTPAttempt(
